client/adjudicate.go

```diff
@@ -167,38 +168,62 @@ func (c *Channel) ProgressBy(ctx context.Context, update func(*channel.State)) e
 	return errors.WithMessage(c.adjudicator.Progress(ctx, *pr), "progressing")
 }
 
-// Settle concludes a channel and withdraws the funds.
+// Settle concludes the channel and withdraws the funds.
 //
 // Returns TxTimedoutError when the program times out waiting for a transaction
 // to be mined.
 // Returns ChainNotReachableError if the connection to the blockchain network
 // fails when sending a transaction to / reading from the blockchain.
```

Comment:
Very good point. We should emphasize more on documentation in the reviews.

Reply:
There is one additional note on that and on documentation in general:
I feel there is a trade off between having a detailed documentation and having a concise one.
A detailed one which goes into the internals is typically much harder to maintain. Because if you change the internals somewhere, you also need to track down all the parts of the documentation that touch upon it, which is easy to do wrong. This is why I usually try to keep documentation short and concise.